### DISCUSSION TITLE
Feature suggestions for tab bar.

### DISCUSSION THREAD

**Discussion by big-edd** [usebruno/bruno#3070] [Ideas]
Hello, 

A few feature suggestions for tabs. 

1.  An option to have the tab bar wrap multiple lines of tabs. 
2.  An option to have the tab bar vertically oriented and scrollable. 

Cheers,
Eddie.

**Comment by ArbiterOfBuffoonery** [comment]
I have a couple of additional tab bar suggestions, let me know if this should be a different discussion.

It would be nice to rearrange the tabs once they are open. If I open 2 requests in a collection the order of the tabs is the order they were opened, and I cannot rearrange them.

Also, it would be nice to have an option to hide the close tab button...I often click it by mistake when selecting a tab with the mouse.
